scormxblock/scormxblock.py

```diff
@@ -196,7 +198,7 @@ def _delete_existing_files(self, path):
         Recusively delete all files under given path
         """
         dir_names, file_names = default_storage.listdir(path)
-        with concurrent.futures.ThreadPoolExecutor(max_workers=1000) as executor:
+        with concurrent.futures.ThreadPoolExecutor(max_workers=10) as executor:
```

Comment:
What's your thought about making it as a constant `MAX_WORKERS` and then use that constant on both places?